frontend/public/components/RBAC/bindings.jsx

```diff
@@ -367,7 +367,7 @@ export const RoleBindingsPage = ({
       <ListPageBody>
         <ListPageFilter
           data={staticData}
-          loaded={loaded}
+          loaded={loaded && resources.ClusterRoleBinding.loaded}
```

Comment:
Is there still potentially a problem if you select namespace bindings and reload the page? I'm wondering if we shouldn't simply change the meaning of `loaded` until everything is loaded even if it takes slightly longer for the table to appear.

Reply:
I agree we should show items once we know enough to show the correct filtering.  There is code to show the table if there is at least one item so I was wondering if performance was an issue.  Of course, showing incorrect information quickly isn't a great option either ;-)

Strangely enough, right now, items not showing on refresh is only an issue for the `Cluster-wide RoleBindings` option.  Here's why.  The items shown in the filter menu  list is dependent on a "hasCRBindings" variable: https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/bindings.jsx#L336-L348.  The `Cluster-wide RoleBindings` is the only option that is different between the two filter menu lists.  

When the page loads and the items are initially filtered, the `hasCRBindings` variable is (or at least can be) false,  and the filter list without `Cluster-wide RoleBindings`  is loaded, therefore it is ignored.

I'll make the change so we wait until items are loaded before we show the table, just wanted to give future us an explanation if needed.